templates/data/kafka/managed.html

```diff
@@ -0,0 +1,330 @@
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-6">
+        <hr class="p-rule--highlight u-no-margin--bottom" />
+        <div class="p-card--overlay u-no-padding--top" style="padding: 1.5rem;">
+          <div class="row p-section--shallow">
+            <div class="col-3 col-medium-3 col-small-2">
+              <h3>Managed Kafka on Azure</h3>
+            </div>
+            <div class="col-3 col-medium-3 col-small-2">
+              <p class="p-heading--3 u-align-text--right u-no-margin--bottom">$3099</p>
+              <p class="u-text--muted u-align-text--right u-no-margin--bottom">
+                Per year,
+                <br />
+                per VM <span aria-labelledby="Management fee does include the cloud infrastructure cost.">*</span>
+              </p>
+            </div>
+          </div>
+          <div class="p-section--shallow">
+            <p class="u-text--muted">The low cost solution for maximum scalability.</p>
+          </div>
+          <hr class="p-rule--muted u-no-margin--bottom" />
+          <div class="p-section--shallow">
+            <ul class="p-list--divided">
+              <li class="p-list__item is-ticked">Highly automated service delivery for security and efficiency</li>
+              <li class="p-list__item is-ticked">Real operations staff proactively monitoring and on call 24/7</li>
+              <li class="p-list__item is-ticked">Includes upgrades and security patches</li>
+            </ul>
+          </div>
+          <div class="p-strip u-no-padding--bottom">
+            <hr class="p-rule--muted" />
+            <p aria-hidden="true" class="u-text--muted u-no-padding--top u-sv2">
+              * Management fee does include the cloud infrastructure cost.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="col-6">
+        <hr class="p-rule--highlight u-no-margin--bottom" />
+        <div class="p-card--overlay u-no-padding--top" style="padding: 1.5rem;">
+          <div class="row">
+            <div class="col-3 col-medium-3 col-small-2">
+              <h3>Managed Kafka on OpenStack</h3>
+            </div>
+            <div class="col-3 col-medium-3 col-small-2">
+              <p class="p-heading--3 u-align-text--right u-no-margin--bottom">$6861</p>
+              <p class="u-text--muted u-align-text--right">
+                Per year,
+                <br />
+                per hypervisor <span aria-labelledby="All hypervisors in the cluster must be covered. Per-VM pricing is also available">*</span>
+              </p>
+            </div>
+          </div>
+          <p class="u-text--muted u-sv1">
+            Secure, managed service delivered to your on-premise data centre by our Kafka operations team.
+          </p>
+          <hr class="p-rule--muted u-no-margin--bottom" />
+          <ul class="p-list--divided u-sv1">
```

Comment:
This `ul` is wrapped in `p-section--shallow` in the Figma design

Reply:
That would misalign hrs on the cards 
![image](https://github.com/user-attachments/assets/9fcf880f-2324-4ae1-98cb-ba56fb7b262b)